flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/ZKCheckpointIDCounterMultiServersTest.java

```diff
@@ -53,7 +53,8 @@ public void testRecoveredAfterConnectionLoss() throws Exception {
         final Configuration configuration = new Configuration();
         configuration.setString(
                 HighAvailabilityOptions.HA_ZOOKEEPER_QUORUM, zooKeeperResource.getConnectString());
-        final CuratorFramework client = ZooKeeperUtils.startCuratorFramework(configuration);
+        final CuratorFramework client =
+                ZooKeeperUtils.startCuratorFramework(configuration, exception -> {});
```

Comment:
Maybe we add something like a `TestingFatalErrorHandler.IGNORE` that is effectively `exception -> {}`.

Reply:
There is already the `NoOpFatalErrorHandler` that could be changed to be a top-level class.